ReactSkia/MountingManager.cpp

```diff
@@ -105,7 +122,7 @@ void MountingManager::CreateMountInstruction(
   auto provider = GetProvider(mutation.newChildShadowView);
   if (provider) {
     std::shared_ptr<RSkComponent> component =
-        provider->CreateAndAddComponent(mutation.newChildShadowView);
+        provider->CreateAndAddComponent(mutation.newChildShadowView,surface_);
```

Comment:
Will need to work on a different way to schedule display update on compositor of RNS SHELL